anthos-bm-gcp-terraform/README.md

```diff
@@ -8,7 +8,7 @@ This repository shows you how to use Terraform to try Anthos clusters on bare me
 - A workstation with access to internet _(i.e. Google Cloud APIs)_ with the following installed
   - [Git](https://www.atlassian.com/git/tutorials/install-git)
   - [Google Cloud SDK](https://cloud.google.com/sdk/docs/install)
-  - [Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli) (~v0.15.5)
+  - [Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli) (>= v0.15.5)
```

Comment:
Can you update the versions here as well please!

Reply:
@Shabirmean done